docker-extension/README.md

```diff
@@ -0,0 +1,7 @@
+## Requirements
+- Docker Desktop (>4.8.0)
+
+## Steps to build and install Headlamp docker extension
+- Clone this repository.
+- Run `docker build -t headlamp ./docker-extension`.
+- Install the extension `docker extension install headlamp`.
```

Comment:
After this, is there somewhere the user can go to run the docker desktop headlamp?

(Sorry if it's obvious... I haven't installed docker desktop yet... will do).